### PR TITLE
bugfix: don't show scope completions for multi-imports

### PR DESCRIPTION
**Pull request by kasiaMarek**
resolves: https://github.com/scalameta/metals/issues/5991

### CODE REVIEW

File: mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala [hunk 1]
```diff
@@ -58,6 +60,7 @@ object CompletionPos:
     val kind =
       if query.isEmpty && !prevIsDot then CompletionKind.Empty
```
Comment (jkciesluk) [inline mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala:61]: I believe we should change this, so `import scala.collection.{AbstractMap, @@}` will work

File: mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala [hunk 2]
```diff
@@ -92,6 +95,19 @@ object CompletionPos:
...
   end inferIndent
 
+  private def isImportSelect(pos: SourcePosition, path: List[Tree])(using
+      Context
+  ): Boolean =
+    @tailrec
+    def loop(enclosing: List[Tree]): Boolean =
+      enclosing match
+        case head :: tl if !head.sourcePos.contains(pos) => loop(tl)
+        case Import(_, sel) :: _ =>
```
Comment (tgodzik) [inline mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala:105]: Should this work in Export also?

File: mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala (hunk 1 above)
Comment (tgodzik) [reply mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala:61]: Where do we check it? Or do you mean we should check it here?

Comment (jkciesluk) [reply mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala:61]: Yes, I mean we should check it here

**Review by jkciesluk — APPROVED** [review]
lgtm

**Review by tgodzik — APPROVED** [review]
LGTM!

**Pull request merged**